### PR TITLE
Zustand home

### PR DESCRIPTION
**Pull request by SaraVieira**
Adds zustand homepage: https://website-git-zustand-home-pmndrs.vercel.app/zustand

Would appreciate some Typoescript help

### CODE REVIEW

**Comment by drcmda** [comment]
something off with the colors and interaction: https://codesandbox.io/s/zustand-site-gpioq

color is because of `<Canvas linear` and interaction i think the easiest is to do pointer-events: none on the overlay and point-events: all on the buttons, links and dialogs.

File: css/main.css [hunk 1]
```diff
@@ -188,3 +188,90 @@ ul.grid-list > li:before {
 .search-item.focus-visible li > div {
   @apply bg-gray-800 text-gray-200;
 }
+
+.zustand-home .bg {
+  background-image: url('https://uploads.codesandbox.io/uploads/user/b3e56831-8b98-4fee-b941-0e27f39883ab/DWLR-bg.jpg');
```
Comment (CodyJasonBennett) [inline css/main.css:193]: We're already hosting these images. Is this outdated or should we point to them?

**Review by CodyJasonBennett — CHANGES_REQUESTED** [review]

File: css/main.css (hunk 1 above)
Comment (SaraVieira) [reply css/main.css:193]: oh crap, will update 


Thanks!

Comment (SaraVieira) [reply css/main.css:193]: This seems like its not even being used anymore

Comment (SaraVieira) [reply css/main.css:193]: Deleted here: https://github.com/pmndrs/website/commit/2517d46b094bbc000455bdf9959fe689ef846b6f

**Pull request merged**